### PR TITLE
bump readout v2.1.0

### PR DESCRIPTION
**Pull request by sy-c**

### CODE REVIEW

File: readout.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package: Readout
 version: "%(tag_basename)s"
-tag: v2.0.2
+tag: v2.1,0
```
Comment (kostorr) [inline readout.sh:3]: There's a comma instead of a dot here ;)

Comment (sy-c) [reply readout.sh:3]: indeed, thanks !

**Review by kostorr — DISMISSED** [review]

**Comment by ktf** [comment]
I think you are missing an explicit cast to `void*`...

**Comment by sy-c** [comment]
> I think you are missing an explicit cast to `void*`...

@ktf I am tired of loosing time "fixing" things blocking when doing PR and that are not even reported as a warning when building with aliBuild on my straight CC7 system.

**Comment by ktf** [comment]
I fully sympathise, but it's not my fault nor alibuild. ;-) As I think I said already the issue is in the readout.sh recipe:

https://github.com/alisw/alidist/blob/master/readout.sh#L38-L46

Since no one did so far, I opened:

https://github.com/alisw/alidist/pull/3022

but someone from FLP should approve / merge it, since my understanding is that this is actually what is requested.

**Comment by awegrzyn** [comment]
@sy-c I'd recommend using GCC 9 from devtoolset.

This warning is generated because in Readout's CMake `-Wall` is enabled, but indeed GCC 7 from devtoolset does not report it.

**Comment by sy-c** [comment]
> 
> 
> @sy-c I'd recommend using GCC 9 from devtoolset.
> 
> This warning is generated because in Readout's CMake `-Wall` is enabled, but indeed GCC 7 from devtoolset does not report it.

Please document this. I'm using the build system exactly as advertised in the documentation.

**Comment by sy-c** [comment]
> 
> 
> I fully sympathise, but it's not my fault nor alibuild. ;-) As I think I said already the issue is in the readout.sh recipe:
> 
> https://github.com/alisw/alidist/blob/master/readout.sh#L38-L46
> 
> Since no one did so far, I opened:
> 
> #3022
> 
> but someone from FLP should approve / merge it, since my understanding is that this is actually what is requested.

@ktf thanks, I was not aware of this issue about the recipes.

**Comment by awegrzyn** [comment]
> Please document this.

GCC 9 is not officially supported but this is what I'm doing.
The actual problem is that GCC 7 from devtoolset does not report the warning but aliBuild compiled one does.

**Comment by sy-c** [comment]
@alisw/infrastructure-admins the CI test for CentOS 8 failed with an unrelated error. Can you please check?

**Review by vascobarroso — APPROVED** [review]

**Pull request merged**